api/pkg/db/model/model.go

```diff
@@ -29,7 +29,7 @@ type (
 		gorm.Model
 		Name       string `gorm:"size:100;not null;unique"`
 		Category   Category
-		CategoryID int
+		CategoryID uint
```

Comment:
does this need any migration changes? I guess not but could you please confirm?

Reply:
Ideally yes it needs a migration but it won't be breaking anything because `CategoryID` refers to Tag `Id` which is the primary key and autogenerated and won't be a signed int. And we don't access category Id except for preloading.

Reply:
but yes will write a migration anyway. Thanks for reminding :)

Reply:
I checked this one, queries are the same for `uint` and `int` as Postgres have the `integer` data type. so, skipping the migration.